inginious/frontend/templates/task.html

```diff
@@ -383,3 +393,21 @@ <h4 class="modal-title" id="socialModalLabel">$:_("Share my result on: ")</h4>
     init_task_page("$task.get_evaluate()");
 });
 </script>
+
+$# Input random
+$# set up some useful variable that may be used by task author
+<script>
+    var __input_random_list = $input_random_list
+    var __username = "$user_manager.session_username()"
+    var __lang = "$user_manager.session_language()"
+    //Shortcut to replace some elements of the page.
+    function replace(old_str, new_str){
```

Comment:
Actually I would merge my two previous comments : we'd let the user specify everything he wants through the ``..raw:: html`` directive and parse ``@@@field@@``-like strings along with the RST, or something else as long as it does not break rst parsing. What do you think ? 

This would require passing the fields dictionary to the rst parse function. The latter may be called outside of the frontend where data like username, lang, etc. have no sense. In this case we can pass an empty dict. 

Actually, the only way to access task data in the future without a web browser should be via the webapp API, which ensures the fields are defined.